chips/fe310-g002/qemu.sh

```diff
@@ -0,0 +1 @@
+qemu-system-riscv32 -M sifive_u,msel=1 -nographic -serial mon:stdio -device loader,addr=0x20010000,cpu-num=0,file=final.bin -semihosting -semihosting-config enable=on,userspace=on
```

Comment:
For my learning: Does a chip (e.g. fe310-g002) always map to a single machine model(e.g. sifive_u) in qemu?

Reply:
Probably. The chip is primarily used to determine the availability, base address, and size of MMIO peripherals, so I doubt we'd be seeing many situations where multiple QEMU machines would work out of the box for the same chip.

Reply:
In that case chip<->machine model can be considered as 1:1. I just wanted to ensure that the terminologies used in hubris and qemu match i.e. chip in hubris is the same as machine model in qemu.

Reply:
I think that every chip should only have 1 machine model that represents it, but multiple chips may share a machine model.  However, there is probably an exception I don't know about.